scss/components/button.scss

```diff
@@ -76,7 +76,11 @@ a.#{$block} {
       }
       &:empty {
         padding: 0;
-        &::before,
+        &::before {
+          margin-right: 0;
+          margin-left: 0;
+          margin-bottom: 1px;
+        }
```

Comment:
I'm curious ... when would a button be "empty"?

Reply:
@greg-a-smith  An empty button is when there is no text between opening and closing tags. 

ie: 
Normal button: `<button class="fd-button--emphasized sap-icon--cart">Add to Cart</button>`
Empty button: `<button class="fd-button sap-icon--cart"></button>`